test-vectors/json/zip_0032_arbitrary.json

```diff
@@ -0,0 +1,8 @@
+[
+    ["From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/zip_0032_arbitrary.py"],
+    ["sk, c"],
```

Comment:
For the other test vectors I think we normally include the input (in this case the context, and the account id in the case of account-specific key derivation), not just the output. For example see https://github.com/zcash/zcash-test-vectors/blob/master/test-vectors/rust/f4jumble.rs or https://github.com/zcash/zcash-test-vectors/blob/master/test-vectors/rust/orchard_group_hash.rs .

Reply:
These test vectors follow how the existing ZIP 32 test vectors work for Sapling and Orchard.